### DISCUSSION TITLE
how do i copy android dependencies on local maven repo with this ./scripts/install-dependencies.sh

### DISCUSSION THREAD

**Discussion by RohanMishra315** [robolectric/robolectric#7914] [Q&A] [answered]
@hoisie hey could you help me how do i copy android dependencies on local maven repo with this  ./scripts/install-dependencies.rb
where to find local maven repo??

**Comment by utzcoz** [comment, marked as answer]
@RohanMishra315 Please don't post the same comment to multiple discussion thread, and we might think it might be spam. As I know, `install-dependencies.sh` is not used for normal contributors. I suppose you know it from `CONTRIBUTING.md`, but this file is outdated, and we will update it when we have time. If you just want to download existed android-all jar files to your local maven repo, you can write a script to download them like normal maven dependencies. I have a simple project to download these dependencies with shell: https://github.com/utzcoz/robolectric-android-all-fetcher. Or you can follow https://cs.android.com/android/platform/superproject/+/master:prebuilts/misc/common/robolectric/update-script/ to download these dependencies with Gradle script.

And you can find local maven repo path by following https://www.baeldung.com/maven-local-repository.

**Reply by Aqil-Ahmad** [reply]
Hi @utzcoz! 
I am also having the same problem, could you please explain that how can we install the dependencies from the links that you provided: https://github.com/utzcoz/robolectric-android-all-fetcher and https://cs.android.com/android/platform/superproject/+/master:prebuilts/misc/common/robolectric/update-script/
